finetune_adapter.py

```diff
@@ -27,7 +27,7 @@
 gradient_accumulation_steps = batch_size // micro_batch_size
```

Comment:
We should add an assertion that `gradient_accumulation_steps != 0`. If `gradient_accumulation_steps == 0`, then there will be an error later in the code in the training loop since you can't do modulo 0.